server/plugin.go

```diff
@@ -162,7 +162,10 @@ func (p *Plugin) handleAdd(w http.ResponseWriter, r *http.Request) {
 
 	receiverMessage := fmt.Sprintf("You have received a new Todo from @%s", senderName)
 	p.sendRefreshEvent(receiver.Id)
-	p.PostBotCustomDM(receiver.Id, receiverMessage, addRequest.Message, issueID)
+	err = p.PostBotCustomDM(receiver.Id, receiverMessage, addRequest.Message, issueID)
```

Comment:
0/5 It might be worth to change the signature of `PostBotCustomDM` to not return an `error` and use it as a fire and forget method, because the caller won't block or return on an `error` anyway.

Reply:
You are right, the error was ignored before these changes. I changed the signature